client/components/Home/CalViews/CalRow.jsx

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable react/prop-types */
+import React, { useContext } from 'react';
+import AppContext from '../../../context';
+
+
+const CalRow = (props) => {
+  const { time, owner, genre, distance } = props.event;
+  const dateArr = time.toString().split(' ');
+  const eventDay = dateArr.slice(0, 3).join(' ');
+  const currentUser = 'Keanu';
```

Comment:
are you using current User anywhere? I can't find it, but tired and brain === dead.